Main.elm

```diff
@@ -1,57 +1,475 @@
+module Main exposing (main)
+
 import Html exposing (..)
```

Comment:
In fact, I didn't really work on the optimisation of my code.
I have prefered to work on the fonctionment  of the application.